includes/admin/class-sensei-setup-wizard.php

```diff
@@ -21,6 +21,7 @@
 class Sensei_Setup_Wizard {
 	const SUGGEST_SETUP_WIZARD_OPTION = 'sensei_suggest_setup_wizard';
 	const WC_INFORMATION_TRANSIENT    = 'sensei_woocommerce_plugin_information';
+	const WCCOM_INSTALLING            = 'sensei_setup_wizard_wccom_installing';
```

Comment:
Does it make sense to add a suffix `_TRANSIENT` to identify the const purpose?

Reply:
Good idea! aa8b2e5